maintainer/FLAKES.md

```diff
@@ -0,0 +1,137 @@
+
+# Finding, reporting and fixing flakes in Envoy's CI
+
+Flakes cause Pull Requests, builds, and release CI to fail, wasting a large
+amount of developer time and CI cycles.
+
+When CI is sufficiently flakey, it becomes easy to ignore new ones, and hard
+to debug real fails.
+
+More often than not, the solution will be reasonably obvious to the author of
+the PR that has introduced the flake, so the quicker this can be brought to their
+attention, the more likely it is that it can be fixed easily.
+
+## Types of flake
+
+### Tests marked flaky
+
+Envoy's bazel has `--flaky_test_attempts` set for much of its CI.
+
+These flakes are not so common, but are reported to the #flakey-test channel in Slack.
```

Comment:
this is not currently correct - but i think will be with https://github.com/envoyproxy/envoy/pull/25624 (fwiw)

Reply:
and the channel is #test-flaky

Reply:
still not quite working, not sure why